zScanner/ViewModels/NewDocument/NewDocumentTypeViewModel.swift

```diff
@@ -38,22 +36,16 @@ class NewDocumentTypeViewModel {
     }
     
     // MARK: Helpers
-    private func fields(for mode: DocumentMode) -> [FormField] {
+    private func getFields() -> [FormField] {
```

Comment:
getFields is not the best name. convert it into variable `fields: [FormField]` :) you don't need the parameter anymore, so it will be nicer